src/app/bugs/bugs-routing.module.ts

```diff
@@ -1,11 +1,24 @@
-import { NgModule } from '@angular/core';
-import { Routes, RouterModule } from '@angular/router';
-import { BugsDashboardComponent } from './components/bugs-dashboard/bugs-dashboard.component';
-
-const routes: Routes = [{path: 'bugsdashboard', component: BugsDashboardComponent}];
-
-@NgModule({
-  imports: [RouterModule.forChild(routes)],
-  exports: [RouterModule]
-})
-export class BugsRoutingModule { }
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { BugsMenuComponent } from './components/bugs-menu/bugs-menu.component';
+import { BugsDashboardComponent } from './components/bugs-dashboard/bugs-dashboard.component';
+import { BugsTechnologyComponent } from './components/bugs-technology/bugs-technology.component';
+
+const routes: Routes = [
+  {
+    path: "",
+    component: BugsMenuComponent,
+    children: [ 
+              {path: '', component: BugsDashboardComponent},
+                {path: 'bugsdashboard', component: BugsDashboardComponent}, 
+                {path: "bugstechnology", component: BugsTechnologyComponent}
+              ]
+  }
+];
+
+@NgModule({
+  imports: [RouterModule.forChild(routes)],
+  exports: [RouterModule]
+})
+export class BugsRoutingModule { };
+export const BugsRoutingComponents = [BugsMenuComponent, BugsDashboardComponent, BugsTechnologyComponent];

```

Comment:
Lets not export this here, rather import this components in the bug-module and add it in the declarations

Reply:
I think export is must here, without export I can;t import in bugs module.

Reply:
you can import the each component in the module, rather than exporting it from here and then again importing.

Reply:
yup the code is updated as you mentioned

Reply:
modified as per your suggestion, please check.